src/core/variable.jl

```diff
@@ -911,9 +830,13 @@ function variable_mc_generation_reactive(pm::_PMs.AbstractPowerModel; nw::Int=pm
     )
 
     if bounded
-        for (i,gen) in _PMs.ref(pm, nw, :gen), c in cnds
-            JuMP.set_lower_bound(qg[i][c], gen["qmin"][c])
-            JuMP.set_upper_bound(qg[i][c], gen["qmax"][c])
+        for (i,gen) in _PMs.ref(pm, nw, :gen)
+            if haskey(gen, "qmin")
+                JuMP.set_lower_bound.(qg[i], gen["qmin"])
+            end
+            if haskey(gen, "qmax")
+                JuMP.set_upper_bound.(qg[i], gen["qmax"])
```

Comment:
should we instead be using the internal `set_lower_bound` and `set_upper_bound` here?